libraries/chain/event_group_evaluator.cpp

```diff
@@ -32,10 +33,30 @@
 
 namespace graphene { namespace chain {
 
+namespace 
+{
+   /// searches for a manager in event_group => sport
+   bool is_manager( const database& db, const event_group_id_type& event_group_id, const account_id_type& manager_id )
```

Comment:
We do have that test in event_evaluator.cpp already, no?

Reply:
This is correct. The test for `is_manager( sport_id )` is also twice ( one in `sport_evaluator` and one in `event_group_evaluator` ). 

The problem is that this methods are used by every evaluator ( `sport_create`, `sport_update` and `sport_delete`, they are helper methods. To not declare them for every `evaluator` I made them static methods to be only available in the `*_evaluator.cpp` files. And because they are declared in the anonymous namespace they have to be declared for every `evaluator` file ( `sport_evaluator.cpp`, `event_group_evaluator.cpp` and `event_evaluator.cpp`.  If I don't declare them in the anon namespace on compilation, they have the same signature as the methods in the other files. The code isn't compiling then.

They could be declared in the `*_evaluator.hpp` and passed to the other `evaluators` down in the hierarchy. Could surely do that. This would result in less duplicate code.